README.md

```diff
@@ -303,40 +324,44 @@ print(naval_chat_bot.query("What unique capacity does Naval argue humans possess
 ```
 
 ### Configs
+
 This section describes all possible config options.
 
 #### **InitConfig**
-|option|description|type|default|
-|---|---|---|---|
-|ef|embedding function|chromadb.utils.embedding_functions|{text-embedding-ada-002}|
-|db|vector database (experimental)|BaseVectorDB|ChromaDB|
+
+| option | description                    | type                               | default                  |
+| ------ | ------------------------------ | ---------------------------------- | ------------------------ |
+| ef     | embedding function             | chromadb.utils.embedding_functions | {text-embedding-ada-002} |
+| db     | vector database (experimental) | BaseVectorDB                       | ChromaDB                 |
 
 #### **Add Config**
 
-*coming soon*
+_coming soon_
 
 #### **Query Config**
 
-|option|description|type|default|
-|---|---|---|---|
-|template|custom template for prompt|Template|Template("Use the following pieces of context to answer the query at the end. If you don't know the answer, just say that you don't know, don't try to make up an answer. \$context Query: $query Helpful Answer:")|
+| option   | description                | type     | default                                                                                                                                                                                                             |
+| -------- | -------------------------- | -------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
```

Comment:
this is hard to maintain, and looks weird in smaller windows. I'd ditch it. But it's up to you.